### PR TITLE
[MINI-6120] Add deprecated annotations for the old architecture

### PR DESCRIPTION
**Pull request by GondiTilak**
# Description
Added deprecated annotation for the methods in MiniApp class which will be removed from SDK in next releases.
 
# Note: 
Some of the methods in this class are called from the demoApp even with the new architecture changes hence must be looked into moving the logic to MiniAppViewHandler or use the apis from hander of new architecture code.

## Links
[MINI-6120](https://jira.rakuten-it.com/jira/browse/MINI-6120)

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [ ] I removed all sensitive data before every commit, including API endpoints and keys
- [ ] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by rleojoseph — COMMENTED** [review]
Could you please update the Readme and Userguide for these changes ?

**Comment by GondiTilak** [comment]
> 

Thank you, I have updated the changelog and user guide as required. 👍🏽

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ### 5.3.0 (XXXX-XX-XX)
 **SDK**
+- **Depricated:** `create(appInfo: ,queryParams: ,completionHandler: ,messageInterface: ,adsDisplayer: ,fromCache: ) in `MiniApp` class, `MiniAppView.load(fromCache: , completion: )` must be used instead.
```
Comment (rleojoseph) [inline CHANGELOG.md:5]:
```suggestion
- **Deprecated:** `create(appInfo: ,queryParams: ,completionHandler: ,messageInterface: ,adsDisplayer: ,fromCache: ) in `MiniApp` class.
```

**Review by rleojoseph — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**